examples/dc/interface/main.tf

```diff
@@ -1,7 +1,15 @@
+# Connect networks to the router
 resource "vkcs_dc_interface" "dc_interface" {
-  name = "tf-example"
-  description = "tf-example-description"
+  name         = "tf-example"
+  description  = "tf-example-description"
   dc_router_id = vkcs_dc_router.dc_router.id
-  network_id = vkcs_networking_network.app.id
-  subnet_id = vkcs_networking_subnet.app.id
+  network_id   = vkcs_networking_network.app.id
+  subnet_id    = vkcs_networking_subnet.app.id
+}
+
+# Connect internet to the router
+resource "vkcs_dc_interface" "dc_interface_internet" {
```

Comment:
this should be a separate file including in its own section in example chapter of doc template

Reply:
done